voxblox_ros/include/voxblox_ros/intensity_vis.h

```diff
@@ -0,0 +1,42 @@
+#ifndef VOXBLOX_ROS_THERMAL_VIS_H_
+#define VOXBLOX_ROS_THERMAL_VIS_H_
+
+#include <voxblox/utils/color_maps.h>
+#include <voxblox_msgs/Mesh.h>
+
+#include "voxblox_ros/conversions.h"
+#include "voxblox_ros/mesh_vis.h"
+
+namespace voxblox {
+
+inline void recolorVoxbloxMeshMsgByIntensity(
```

Comment:
Is there a reason why is the color not applied at a lower level, i.e. to the voxblox::Mesh ?

Reply:
Easier to do it, I guess? Fewer problems with consistency when updating the mesh incrementally and you don't have to keep 2 full copies around?